src/V3Assert.cpp

```diff
@@ -45,6 +45,7 @@ class AssertVisitor final : public VNVisitor {
     VDouble0 m_statAsNotImm;  // Statistic tracking
     VDouble0 m_statAsImm;  // Statistic tracking
     VDouble0 m_statAsFull;  // Statistic tracking
+    bool m_inSampled = false;
```

Comment:
Need comment on every variable please (even when perhaps obvious)

Reply:
Okay, I've added the lacking comment (ace7feec57b91cdd5d872bb346c0da8a7e1316c2).